src/main/java/com/gmail/val59000mc/languages/Lang.java

```diff
@@ -331,6 +332,7 @@ private void loadLangConfig() {
 
 		// Event
 		EVENT_TIME_REWARD = getString(lang, "event.time-reward", "&eYou have received %money% in your account for playing %time% , total playing time %totaltime%");
+		EVENT_TIME_COMMANDS = getString(lang, "event.time-commands", "&eYou successfully survived %time%, total playing time %totaltime%");
```

Comment:
Why did you add this message for `EVENT_TIME` but not for `EVENT_KILL`? Also the message doesn't mention anything about the commands getting executed so I don't really see why it would be needed either. If people really want a message they can use a command for that.